frontend/pages/recipe/create.vue

```diff
@@ -42,6 +42,11 @@ export default defineComponent({
         text: "Import with URL",
         value: "url",
       },
+      {
+        icon: $globals.icons.fileImage,
+        text: "Create recipe from an image",
```

Comment:
Should be title cased like the rest of the menu, can you move this below "Import With .zip"